### ISSUE TITLE
add-fdroid-repo page query

### ISSUE THREAD

**Issue by yashpalgoyal1304** [TeamNewPipe/website#266]
[adding repo](https://newpipe.net/FAQ/tutorials/install-add-fdroid-repo/)[..](https://github.com/TeamNewPipe/website/blob/master/_tutorials/install-add-fdroid-repo.html) says to
* "click" the link 
* i think it was supposed that it'd open with fdroid
* but it doesnt happen, instead the page for `Index of /fdroid/repo/` is opened
![image](https://user-images.githubusercontent.com/19423063/145065565-ca7b99b9-f0b9-4928-8331-f8488ea54f37.png)

The only way i can think of to make it work is to open it with intent redirector app like [Open Link with](https://f-droid.org/packages/com.tasomaniac.openwith.floss/) 

So, how is it supposed to work otherwise? is it intended? or does the wording need to change there?

**Comment by yashpalgoyal1304** [comment]
so, i have tested on 3-4 devices (running android 10, 11), and the issue seems to be with the app from which the above link is clicked. 

The behaviours i'd categorise as 2:
* vv means 
app picker shown, f-droid present there
* xx means 
no app picker shown (and proceeds to the webpage which shows above behav)


| Behaviour | App Category | Apps |
| -- | -- | -- |
| xx | Browsers | Vivaldi 4.1.2366.26 <br/> Chrome 87.0.4280.101 |
| xx | Browser's that isolated tab like thingy | Vivaldi <br/> Chrome |
| vv | Browsers | [LAOS Jelly] v1.0
| vv  | Other apps | Telegram 8.2.7 <br/> [QR Scanner] 1.9 <br/> HTML viewer (via .MD file) |

[LAOS Jelly]: https://www.apkmirror.com/app/org.lineageos.jelly
[QR Scanner]: https://f-droid.org/packages/com.example.barcodescanner

Further tests that can be done:
* Newer browsers on old phones
* old browsers on new phones
* testing in webview

-------
So, for my (semi-biased) conclusion:
* i think that google changed browser's behaviour to break the intent parsing/intent interception for external apps/fdroid
* i am saying so, 'cz i have seen google play protect once blocked fdroid, so, fdroid is definitely on google's list
* and this is suuper shady, 'cz the links for play store when clicked from browsers open straight in play store app without even showing the open with/app selector pagelet.

**Comment by yashpalgoyal1304** [comment]
The comment https://github.com/TeamNewPipe/NewPipe/issues/7436#issuecomment-977718002 sheds some light on an issue which mayyyybe related to this, though i tested with android 10/11....

**Comment by yashpalgoyal1304** [comment]
@linsui @licaon-kter any comments??

tl;dr:
browsers are not redirecting repo links to be handled by open with dialog, and so, fdroid is not shown.....
so, something has to be done there regarding this imo: https://f-droid.org/en/tutorials/add-repo/

does this seem good enough to open an issue on fdroid's own repo? i'll copy it there if so.

**Comment by licaon-kter** [comment]
There's an F-Droid issue (in Client iirc) about the way different apps react to a repo link, some open F-Droid...some don't.
